Adds Create, Delete, Retention (#2020)
temporalio/documentation: docs-src/cloud/tcld/namespace/delete.md

---
id: delete
title: tcld namespace delete
sidebar_label: delete
description: How to delete a Namespace in Temporal Cloud using tcld.
tags:
  - tcld
---

The `tcld namespace delete` command deletes the specified [Namespace](/concepts/what-is-a-namespace) in Temporal Cloud.

Alias: `d`

`tcld namespace delete`

The following modifiers control the behavior of the command.

### `--namespace`

> Specify the Namespace hosted on Temporal Cloud to be deleted.

Reviewer: ```suggestion
_Required modifier_

Specify the Namespace hosted on Temporal Cloud to be deleted.
```

Alias: `-n`

### `--request-id`

> The request ID to use for the asynchronous operation. If not set, the server will assign one.

Reviewer: ```suggestion
The request identifier to use for the asynchronous operation.
If not set, the server assigns an identifier.
```

Alias: `-r`

### `--resource-version`

> The resource version (etag) to update from. If not set, the CLI will use the latest.

Reviewer: ```suggestion
A resource version (ETag) to update from.
If not set, the CLI uses the latest.
```
